components/automate-gateway/api/iam/v2/teams.proto

```diff
@@ -58,16 +174,68 @@ service Teams {
     option (chef.automate.api.iam.policy).resource = "iam:teams:{id}";
     option (chef.automate.api.iam.policy).action = "iam:teams:delete";
   };
+
+  /*
+  Gets local team membership
+
+  Lists all users of a local team. Users are listed by their membership_id.
```

Comment:
oh no, it's a membership_id? Even though the response calls it a user_id? 🤦‍♂ Why would we do that? lol

Reply:
Hmm, I'm seeing:
```curl -kH "api-token: $TOK" https://a2-dev.test/apis/iam/v2/teams/editors/users
{"user_ids":["57ff772b-a91d-4ae0-9252-2401ef63153b","viewer","foo"]}

I have no idea what that first one is, but the others are ids I passed in to addusers . . .